app/models/item.rb

```diff
@@ -25,4 +25,20 @@ def no_orders?
     item_orders.empty?
   end
 
+  def total_quantity_ordered
+    item_orders.sum(:quantity)
+  end
+
+  def self.all_active
+    where(active?: true)
+  end
+
+  def self.five_most_popular
+    joins(:item_orders).group("items.id").order("SUM(item_orders.quantity) DESC").take(5)
```

Comment:
Great use of ActiveRecord. I'd like to explore if we can use the `total_quantity_ordered` method within the `order` AR method instead of the SQL injection.

Reply:
I wonder if there are any conventions on using instance methods within class methods...